libp2p/network/connection/raw_connection.py

```diff
@@ -55,4 +55,8 @@ def __init__(
         self.writer.close()
         if sys.version_info < (3, 7):
             return
-        await self.writer.wait_closed()
+        try:
+            await self.writer.wait_closed()
+        # In case the connection is already reset.
+        except (ConnectionResetError, RawConnError):
```

Comment:
The writer is an `asyncio.StreamWriter` type, so we don't need to handle `RawConnError` here right?